hijack/locale/nl/LC_MESSAGES/django.po

```diff
@@ -7,38 +7,41 @@ msgid ""
 msgstr ""
 "Project-Id-Version: \n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2021-03-14 11:27+0100\n"
-"PO-Revision-Date: 2021-03-14 11:35+0100\n"
-"Last-Translator: Johannes Maron <info@johanneshoppe.com>\n"
+"POT-Creation-Date: 2024-04-29 13:23+0200\n"
+"PO-Revision-Date: 2024-04-29 13:24+0200\n"
+"Last-Translator: Johannes Maron <johannes@maron.family>\n"
 "Language-Team: \n"
 "Language: nl\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"X-Generator: Poedit 2.4.2\n"
+"X-Generator: Poedit 3.4.2\n"
 
-#: contrib/admin/admin.py:35
-msgid "hijack user"
-msgstr "gebruiker kapen"
+#: contrib/admin/admin.py:64
+msgid "impersonate user"
+msgstr "zich voordoen als gebruiker"
```

Comment:
I think the verb 'overnemen' (to take over) is better suited for this purpose.

'zich voordoen als' is very literal translation. I'd translate it back to English as 'to pretend to be user'. It doesn't fit well.

```suggestion
msgstr "gebruiker overnemen"
```